packages/engineer/src/cli-commands.ts

```diff
@@ -53,6 +53,7 @@ export const startCommand: Command = (program) =>
             true
         )
         .option('--title <title>', 'application title to display in browser')
+        .option('--favicon <favicon>', 'application favicon to display in browser')
```

Comment:
I think `faviconPath` will be better here
also about the description, maybe `path to favicon to be displayed in browser environments`